docs/reference/part_properties.rst

```diff
@@ -0,0 +1,350 @@
+Part properties
+===============
+
+.. Ideally, this would be automatically generated.
+
+.. _after:
+
+after
+-----
+**Type:** array of unique strings with at least 1 item |br|
+**Step:** build
+
+Specifies a list of parts that a given part will be built *after*.
+
+.. _build_attributes:
+
+build-attributes
+----------------
+**Type:** array of unique strings with at least 1 item from "core22-step-dependencies", "enable-patchelf", "no-patchelf", "no-install", "debug", "keep-execstack". |br|
+**Step:** build
+
+The customisations to apply to the build.
+
+.. _build_environment:
+
+build-environment
+-----------------
+**Type:** build-environment-grammar |br|
+**Step:** build
+
+The environment variables to be defined in the build environment specified as
+a list of key-value pairs.
+
+**Example:**
+
+.. code:: yaml
+
+   build-environment:
+     - MESSAGE: "Hello world"
+     - NAME: "Craft Parts"
+
+.. _build_packages:
+
+build-packages
+--------------
+**Type:** grammar-array |br|
+**Step:** build
+
+The system packages to be installed in the build environment before the build
+is performed. These are installed using the host's native package manager,
+such as :command:`apt` or :command:`dnf`, and they provide libraries and
+executables that the part needs during the build process.
+
+.. _build_snaps:
+
+build-snaps
+-----------
+**Type:** grammar-array |br|
+**Step:** build
+
+The snaps to be installed in the build environment before the build is
+performed. These provide libraries and executables that the part needs during
+the build process.
+
+.. _disable_parallel:
+
+disable-parallel
+----------------
+**Type:** boolean |br|
+**Step:** build
+
+By default, Craft Parts builds independent parts in parallel. This can be
+disabled by setting the ``disable-parallel`` key to ``True``.
```

Comment:
This property is indeed defined in the part model in Craft Parts, but it's not actually used. I think we should let it out of the documentation for now until we confirm if it was defined by mistake and can be removed from the part model (it was used in Snapcraft plugins V1, but it doesn't seem that it exists in plugins V2).

Reply:
I'll remove it.